build.gradle.kts

```diff
@@ -102,6 +104,14 @@ subprojects {
         }
     }
 
+    extensions.findByType(MavenPublishBaseExtension::class)?.apply {
+        publishToMavenCentral(
+            host = SonatypeHost.DEFAULT,
+            automaticRelease = true // instead of closeAndReleaseRepository
```

Comment:
nie mam dowstepu do konfiguracji workflow ale wydaje mi sie (patrzac na readme) ze to powinno zastapic closeAndRelease

Reply:
a tak poza tym to nie wiem co robie bo nie mam jak tego potestowac lokalnie :)